src/app/app.component.css

```diff
@@ -1,11 +1,11 @@
 .main-container {
-  color: #ffffff;
-  margin-top: 61px;
+  color: var(--text-color);
```

Comment:
I would give another name to this var to make it more clear (e.g., white-color, white-text or simply white)